src/test/scala/nl/knaw/dans/lib/logging/servlet/AbstractServletLoggerSpec.scala

```diff
@@ -74,12 +74,11 @@ class AbstractServletLoggerSpec extends FlatSpec with Matchers with EmbeddedJett
 
       val requestLine :: responseLine :: Nil = resultLines
 
-      requestLine should startWith(s"GET http://localhost:$port$path")
+      requestLine should startWith(s"request GET http://localhost:$port$path")
       requestLine should include(s"remote=$formattedRemote;")
 
-      responseLine should startWith(s"GET http://localhost:$port$path returned status=200; ")
+      responseLine should startWith(s"response GET http://localhost:$port$path returned status=200; ")
```

Comment:
I expected something like
`responseLine should include(s"headers=[]")`

Reply:
In these concrete tests in `AbstractServletLoggerSpec`, there are two headers: `Date` and `Content-Type`. The former is (obviously) different for every run and the latter is tested for in the next line.

https://github.com/DANS-KNAW/dans-scala-lib/blob/5a49db9f5eedd86623871a9ba36881ccdb83203f/src/test/scala/nl/knaw/dans/lib/logging/servlet/AbstractServletLoggerSpec.scala#L81

The full response looks something like:

```
response GET http://localhost:65109/combinedLogger returned status=200; headers=[Date -> [Tue, 05 Mar 2019 08:20:23 GMT], Content-Type -> [text/plain;charset=utf-8]]
```

So, the headers are present, I did not remove them, even though it almost looks like it in

https://github.com/DANS-KNAW/dans-scala-lib/pull/17/files#diff-acef33b8123f7db626862068939d5ceaL82

To be sure I changed the test here as well:

```diff
-responseLine should startWith(s"response GET http://localhost:$port$path returned status=200; ")
+responseLine should startWith(s"response GET http://localhost:$port$path returned status=200; headers=[")
```

And similar in `ServletLoggerSpec`:

```diff
(mockedLogger.info(_: String)) expects where {
  s: String =>
-    (s startsWith s"response GET http://localhost:$port$testLoggerPath returned status=200") &&
+    (s startsWith s"response GET http://localhost:$port$testLoggerPath returned status=200; headers=[") &&
      (s.toLowerCase contains "content-type -> [text/plain;charset=utf-8]")
} once()
```

The full log line with merged headers is primarily tested in

https://github.com/DANS-KNAW/dans-scala-lib/pull/17/files#diff-222697fe1f39182d7abe71cfb9fbfc71R57